calliope/test/test_example_models.py

```diff
@@ -354,7 +354,7 @@ def test_urban_example_results_cap_gurobi(self):
 
     def test_milp_example_results(self):
         model = calliope.examples.milp(
-            override_dict={'model.subset_time': '2005-01-01'}
+            override_dict={'model.subset_time': '2005-01-01', 'run.solver_options.mipgap': 0.001}
```

Comment:
why overriding mipgap here?

Reply:
because it takes ages to converge on the LP relaxation, due to the introduction of the binary operation switch